automatic/firefox/README.md

```diff
@@ -17,7 +17,22 @@ Bringing together all kinds of awesomeness to make browsing better for you.
 
 ## Package Parameters
 
-- `l=<locale>` - Install given Firefox locale. For example `choco install Firefox --params "l=en-GB"`. See the [official page](https://releases.mozilla.org/pub/firefox/releases/latest/README.txt) for a complete list of available locales.
+- `l=<locale>` - Install given Firefox locale. See the [official page](https://releases.mozilla.org/pub/firefox/releases/latest/README.txt) for a complete list of available locales.
+
+Command-line options for installer configuration. See the [official page](https://firefox-source-docs.mozilla.org/browser/installer/windows/installer/FullConfig.html) for details and defaults.
+- `/InstallDirectoryPath=<path>`
+- `/InstallDirectoryName=<path>`
+- `/TaskbarShortcut=<true/false>`
+- `/DesktopShortcut=<true/false>`
+- `/StartMenuShortcut=<true/false>`
+- `/MaintenanceService=<true/false>`
+- `/RemoveDistributionDir=<true/false>`
```

Comment:
I would remove this one as it is only for uninstall and doesn't seem important in choco context.

Reply:
This is actually very important and the main reason for this PR. The Distribution directory contains the policies.json file which allows you to configure FF (see https://support.mozilla.org/en-US/kb/customizing-firefox-using-policiesjson). On updating FF this directory get's removed by the installer. I will change it to reflect the always true if exists paradigm.

Reply:
Then it should be always set to false  and eventually add `/RemoveDistributionDir` to enable it ?

Reply:
I totally agree but hesitated to change the defaults. It makes more sense to keep the directory by default, especially in a managed software context.